src/ploomber/executors/parallel.py

```diff
@@ -90,6 +89,30 @@ class Parallel(Executor):
     >>> dag = DAG(executor=Parallel(processes=2)) # customize
 
 
+    DAG can exit gracefully on function tasks (PythonCallable):
+
+    >>> # A PythonCallable function that raises DAGBuildEarlyStop
+    >>> def early_stop_root(product):
+    >>>     raise DAGBuildEarlyStop('Ending gracefully')
```

Comment:
I think this will break, the line that starts with `raise` should begin with `...` instead of `>>>` to be recognized as a [doctest](https://docs.python.org/3/library/doctest.html#how-are-docstring-examples-recognized)

it's better if you run things locally, and then push. when you push we run all the tests and it takes some time to see the results. here's how you can run these doctests:

https://github.com/ploomber/ploomber/blob/cd16d6f7b570015995743fce1c602feeb57b4618/.github/workflows/ci-unit-linux.yml#L88

Reply:
To run all the tests locally (the tests we run when I push) in addition to the doctests mentioned here, do I just run `pytest tests/util` ?